plugins/spigot/src/main/java/com/managemc/spigot/service/ReportsService.java

```diff
@@ -0,0 +1,49 @@
+package com.managemc.spigot.service;
+
+import com.managemc.api.ApiException;
+import com.managemc.api.wrapper.model.metadata.ExternalServerAuthMetadata;
+import com.managemc.spigot.command.util.CommandValidationException;
+import com.managemc.spigot.config.SpigotPluginConfig;
+import org.bukkit.entity.Player;
+import org.openapitools.client.model.CreateWatchRequestInput;
+
+import java.util.UUID;
+
+public class ReportsService {
```

Comment:
This class is for consuming ManageMC's API and handling any specific errors that could be described as "expected". We generally don't do things like send messages to the `CommandSender` here; that has historically gone in the command handler.